externals/kyuubi-flink-sql-engine/src/test/scala/org/apache/kyuubi/engine/flink/operation/FlinkOperationSuite.scala

```diff
@@ -761,4 +761,16 @@ class FlinkOperationSuite extends WithFlinkSQLEngine with HiveJDBCTestHelper {
         .getStringVal.getValues.get(0) === "tmp.hello")
     }
   }
+
+  test("ensure result max rows") {
```

Comment:
Could you please add the test case for setting the `ENGINE_FLINK_MAX_ROWS`?

Reply:
@SteNicholas Sure. Thanks for pointing that out.